y/file_dsync.go

```diff
@@ -1,4 +1,4 @@
-// +build !dragonfly,!freebsd,!windows
+// +build !dragonfly,!freebsd,!windows,!plan9
```

Comment:
Shouldn't `plan9` be present in `y/file_nodsync.go` file? Otherwise the `datasyncFileFlag` would always be 0.

Reply:
Plan 9's `syscall.O_SYNC` has a made up value of 0 (since Plan 9 doesn't support it) but you're right, `plan9` tag should be added to `file_nodsync.go` for consistency.

Reply:
Done